### PR TITLE
Cross references

### PR DESCRIPTION
**Pull request by seanredmond**
Adds `<crossRef>` element and updates more examples to reflect DTD changes.

### CODE REVIEW

**Review by hadro — COMMENTED** [review]
Looks fine, but out of curiousity, what sort of thing do you expect in here?
(Sorry, that was meant to be a comment on just commit 2159935 ...)

File: CopyrightEntries.dtd [hunk 1]
```diff
@@ -30,6 +30,7 @@
 <!ELEMENT newMatterClaimed (#PCDATA)>
 <!ELEMENT claimant (#PCDATA)>
 <!ELEMENT claimantPlace (#PCDATA)>
+<!ELEMENT note (#PCDATA)>
```
Comment (hadro) [inline CopyrightEntries.dtd]: Looks fine, but out of curiousity, what sort of thing do you expect in here?

File: CopyrightEntries.dtd [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
-<!ELEMENT copyrightEntries (copyrightEntry+) >
+<!ELEMENT copyrightEntries (copyrightEntry | crossRef)+ >
 <!ELEMENT copyrightEntry ANY>
 <!ATTLIST copyrightEntry
 	  id NMTOKEN #REQUIRED
 	  regnum NMTOKEN #IMPLIED>
+
+<!ELEMENT crossRef (#PCDATA | author | title | see)*>
+<!ATTLIST crossRef
+	  id NMTOKEN #REQUIRED>
```
Comment (hadro) [inline CopyrightEntries.dtd:10]: 👍, yes, this makes sense to me now

**Review by hadro — COMMENTED** [review]
I took a look, and as you say, this mostly looks like it loosens things up, and looks fine, but as noted previously not my area of expertise. Want to merge and send to DCL folks for review/comment, or want to send it to them first?

**Comment by seanredmond** [comment]
On the question about `<note>`, it would be for any of the bracketed notes in the entries, like:

    <title>Mathematics, queen and servant of science.</title>
    <note>[An integration and amplification of The queen of the sciences (1931) and The handmaiden of the sciences (1937)]</note>

File: CopyrightEntries.dtd (hunk 2 above)
Comment (seanredmond) [reply CopyrightEntries.dtd:10]: Lets merge it first and see what they think at DCL

**Pull request merged**